daisy_tutorials/modify_image/README.md

```diff
@@ -0,0 +1,197 @@
+# Modifying a base image using Daisy
+
+Daisy workflows [have many
+capabilities,](https://github.com/GoogleCloudPlatform/compute-image-tools/tree/master/daisy#workflow-config-overview)
+and it is hard to know where to start. This tutorial will demonstrate
+writing a very simple workflow to modify a GCE base image.
+
+The overall steps of the workflow will be:
+
+1. Create a new instance
+2. Run a provisioning script to modify the instance
+3. Create a new disk image from the instance
+4. Delete the instance
+
+## Provisioning the instance
+
+What you use to provision the instance is open and not dependant on
+Daisy. In this simple case, we are using a simple shell script to
+install Emacs. Because we are creating a CentOS image, we simply call
+`yum install`. The provisioning script is named `emacs_install.sh`.
+
+```shell
+#!/bin/bash
+
+yum -y install emacs-nox
+shutdown -h now
+```
+
+The script shuts down the instance after it is done. We will use this
+later in the Daisy workflow to know when provisioning is done.
+
+## Start writing the workflow
+
+First we will start with a Name:
+
+```
+{
+    "Name": "emacs-image",
+```
+
+Then, list the sources. We will need the provisioning script above.
+
+```
+    "Sources": {
+        "emacs_install.sh": "./emacs_install.sh"
+    },
+```
+
+## Create and Provision the Instance
+
+Now we will start writing steps in the workflow. We will create the
+disk first, as we need to name it and perform operations on it
+later. We give the disk a name, specify the base image, and specify
+the SSD type for speed. The name of the step `create-disks` is
+arbitrary.
+
+```
+    "Steps": {
+        "create-disks": {
+            "CreateDisks": [
+                {
+                    "Name": "instance-disk",
+                    "SourceImage": "projects/centos-cloud/global/images/family/centos-7",
+                    "Type": "pd-ssd"
+                }
+            ]
+        },
+```
+
+Next create the instance using the above disk, and specify the startup
+script that we wrote to provision the instance.
+
+```
+        "create-inst-install": {
+            "CreateInstances": [
+                {
+                    "Name": "inst-install",
+                    "Disks": [{"Source": "instance-disk"}],
+                    "MachineType": "n1-standard-4",
```

Comment:
Probably go with n1-standard-1 here (only because 4 CPU's to install a package is a little bit much and we like to err on the side of using less resources for the example cases).

Reply:
Fixed